app/dao/services_dao.py

```diff
@@ -278,7 +275,7 @@ def dao_create_service(
         raise ValueError("Can't create a service without a user")
 
     if service_permissions is None:
-        service_permissions = DEFAULT_SERVICE_PERMISSIONS
+        service_permissions = ServicePermissionType.defaults()
```

Comment:
why is it a method call here?  Why not ServicePermissionType.DEFAULTS?

Reply:
Because it results in a list of permissions, rather than a single permission.